OutOfSchool/OutOfSchool.BusinessLogic/Services/Elasticsearch/IElasticPinger.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace OutOfSchool.BusinessLogic.Services;
+
+public interface IElasticPinger
```

Comment:
(Naming) Not sure if interface containing the only property - `IsHealthy` - should be named as `IElasticPinger`. Please consider renaming

Reply:
Thanks for renaming. Resolving this comment.

Just for your information let me share few more thoughts without any further call to action. 

The "better" design for this interface with only `IsHealthy` property for specific service would be having 2 interfaces:
- the `IHealthy` interface containing "logic" (`IsHealthy` property)
- and the marker interface - `IHealthyElastic` (empty, implementing `IHealthy` interface) - used for DI, and specifically implemented in class.

Suggested approach is more extensible (when you need to add another service with health check - you would just add another marker interface implementing `IHealthy`), though it's more wordy (for current solution it's 2 interfaces instead of 1). 
Due to latter, I don't insist on any changes, but would like you to know possible options.

Reply:
Thanks! This information is really helpful!